tickit/devices/eiger/eiger_settings.py

```diff
@@ -0,0 +1,135 @@
+from dataclasses import asdict, dataclass, field
+from enum import Enum
+from typing import List, Union
+
+from .eiger_schema import (
+    AccessModes,
+    field_config,
+    ro_float,
+    ro_str,
+    rw_bool,
+    rw_float,
+    rw_int,
+    rw_str,
+)
+
+FRAME_WIDTH: int = 4148
+FRAME_HEIGHT: int = 4362
+
+
+class KA_Energies(Enum):
```

Comment:
Minor nitpick but some of your enum names are plural and some are singular, I'd recommend making them all singular.
```suggestion
class KA_Energy(Enum):
```